x/gamm/keeper/multihop.go

```diff
@@ -52,6 +52,9 @@ func (k Keeper) MultihopSwapExactAmountOut(
 	if err != nil {
 		return sdk.Int{}, err
 	}
+	if len(insExpected) == 0 {
+		return sdk.Int{}, nil
+	}
```

Comment:
Probably belongs in a separate PR but good addition (small enough to keep in this one imo)